### ISSUE TITLE
Handling of coordination+ellipsis

### ISSUE THREAD

**Issue by nikitakit** [UniversalDependencies/UD_Finnish-FTB#2]
The following sentence appears to be a case of clause coordination with an elided verb in the latter two clauses, but the analysis doesn't seem to match with what I can find in the [UD guidelines](https://universaldependencies.org/u/overview/complex-syntax.html). Is this possibly an annotation error, or am I not understanding things right? (Disclaimer: I have no knowledge of Finnish)

![12951_0](https://github.com/UniversalDependencies/UD_Finnish-FTB/assets/252225/21135bef-ffa6-4d97-9d14-810c0d367654)

```
# sent_id = wvlzx-11083.2
# text = Hanna Parviainen maksoi Säynätsalon kirkon 1926 , Walter Ahlström Noormarkun 1929 aloitetun kirkon ja Gösta Serlachius Mäntän kirkon 1928 .
1       Hanna   hanna   PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    3       nsubj   _       Alt=name
2       Parviainen      parviainen      PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    1       flat    _       _
3       maksoi  maksaa  VERB    V,Act,Ind,Past,Sg3      Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin|Voice=Act 0       root    _       _
4       Säynätsalon     säynätsalo      PROPN   N,Prop,Sg,Gen   Case=Gen|Number=Sing    5       nmod    _       _
5       kirkon  kirkko  NOUN    N,Sg,Gen        Case=Gen|Number=Sing    3       obj     _       _
6       1926    1926    NUM     Num,Card        NumType=Card    3       nmod    _       _
7       ,       ,       PUNCT   Pun     _       6       punct   _       _
8       Walter  walter  PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    1       conj    _       Alt=name
9       Ahlström        ahlström        PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    8       flat    _       _
10      Noormarkun      noormarkku      PROPN   N,Prop,Sg,Gen   Case=Gen|Number=Sing    13      nmod    _       _
11      1929    1929    NUM     Num,Card        NumType=Card    12      nmod    _       _
12      aloitetun       aloittaa        VERB    V,Pass,PcpNut,Sg,Gen    Case=Gen|Number=Sing|PartForm=Past|VerbForm=Part|Voice=Pass     13      acl     _       _
13      kirkon  kirkko  NOUN    N,Sg,Gen        Case=Gen|Number=Sing    5       conj    _       _
14      ja      ja      CCONJ   Pcle,CC _       18      cc      _       _
15      Gösta   gösta   PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    1       conj    _       Alt=name
16      Serlachius      serlachius      PROPN   N,Prop,Sg,Nom   Case=Nom|Number=Sing    15      flat    _       _
17      Mäntän  mänttä  PROPN   N,Prop,Sg,Gen   Case=Gen|Number=Sing    18      nmod    _       _
18      kirkon  kirkko  NOUN    N,Sg,Gen        Case=Gen|Number=Sing    5       conj    _       _
19      1928    1928    NUM     Num,Card        NumType=Card    6       conj    _       _
20      .       .       PUNCT   Pun     _       19      punct   _       _
```

**Comment by nikitakit** [comment]
After looking at this some more, it looks like the annotation is consistent in annotating coordination by connecting pairs of words that are "parallel" to each other.

It is still my impression, though, that the UD guidelines and some of the other Finnish treebanks follow a different convention.

At least the following sentences in the training data appear to be affected by this:
- `xezs0-6140`
- `r7bh8-8531`
- `xox7g-9207`
- `v125q-11046`
- `ct5yn-11059`
- `wvlzx-11083.2`
- `tpx7s-11098`
- `fzvdh-11101`
- `xz0c3-15094.1`
- `z46wt-15347`
- `yujti-17090`
- `tyszx-6712` (maybe)

And at least the following sentences in the test set:
- `c1ro6-13800`
- `saw28-14137`
